### ISSUE TITLE
The dialog disappear after click

### ISSUE THREAD

**Issue by za-ek** [charliekassel/vuejs-datepicker#614]
A datepicker component inside <label> tag disappear after click (select year, previous month, etc..)
`<label><datepicker ...></label>` but still work selecting any date in month.

**Comment by charliekassel** [comment]
Yes having the label wrapped around the datepicker causes some strange issues.
I've never really got to the bottom of this so always use the label as a sibling element rather than a parent.

**Comment by paul-mitchell-bjss** [comment]
`<label @click.prevent><datepicker ...></label>`, though not a solution, doesn't suffer from this issue. I think what's happening is the picker allows the click event to bubble so the wrapping label handles it and by default focuses the input which blurs the picker causing it to disappear.
